### DISCUSSION TITLE
Question about Cr10s Pro v1 and tiny machines bltouch kit

### DISCUSSION THREAD

**Discussion by nfored** [Desuuuu/klipper#86] [Q&A]
I found this fork while searching and saw many people use this successfully with the CR10s pro and the example configuration  found here:
https://github.com/Desuuuu/DGUS-reloaded-Klipper-config

Sounds like I just install klipper and use the cr10s pro configuration from above and it should just work.  I do have one question though. Tiny Machines Bltouch v 3.1 kit changes the zprobe configuration.  I see the following in their guide.

Can anyone help me with what I need to change in the klipper configuration to allow this to work?

Thank you kindly for your time and any help provided.
-Fred

Daughter Board instructions: 
Unplug the original probe connector. It may have glue on it so pull on the connector, not the wires.

Mainboard instructions:
3 pin black plug into D11 (end row towards SD card slot) Orange wire will face the ribbon cable like in the picture.
2 pin white plug into Z+ (connection closest to the ribbon cable) From the same view as the picture, use the right 2 pins.

Full instructions 
https://tiny-machines-3d.myshopify.com/products/bl-touch-kit-for-creality-10s-pro

**Comment by Desuuuu** [comment]
This is the same BLTouch setup as a CR-10S Pro V2.

If this is the only hardware change you've made, you can use the configuration made for the V2.

**Reply by nfored** [reply]
I added a a non direct dive microswiss but I think that must be same location as the factory as I never seemed to need to change any offsets. Thank you so much for your help here. Amazing fast response, I work for a very large network company with dozens of developers and If I got a response in 3 days it would be amazing.

**Reply by Desuuuu** [reply]
Upgrading to a MicroSwiss hotend doesn't require any configuration update indeed.

If you'd like, you could adjust the extruder's max_temp to account for it though.
